app/components/documents/document-card.hbs

```diff
@@ -62,6 +62,7 @@
               <div class="auk-o-flex auk-o-flex--vertical-center">
                 <AccessLevelPill
                   @accessLevel={{this.accessLevel}}
+                  @isDraft={{not (await (is-access-level-in-effect this.accessLevel this.piece))}}
```

Comment:
Not convinced this is the best way.
For most of the use cases, we start from a route which has all the info about meeting / agenda / agendaitems but then we go into the document card and now have to get all the info again because the document-card is so generic and knows nothing about the meeting / agenda / agendaitem we started from.

Reply:
Could we callback to the original route to get this needed data?
just pass along the callback down the document-card tree and call it on construction of the component?

Reply:
Passing a callback could help here to make the helper less bulky, but that also means drilling down a prop through a lot of layers of components, although it might be the better option in this case. But I will give it a try (once the data migration stuff is finished). Thanks for the tip